zerver/views/messages.py

```diff
@@ -910,9 +910,12 @@ def update_message_backend(request, user_profile,
         if content == "":
             raise JsonableError(_("Content can't be empty"))
         content = truncate_body(content)
-
+        ums = UserMessage.objects.filter(message=message.id)
```

Comment:
Just checking...I think I agree with you that we want to include the historical=False users, because retroactive alerts can make sense in some contexts, right?  I'm thinking you have an alert word like "support", and maybe a newbie subscribes to support and needs to look at the old messages flagged for "support."  The counterargument is that there's no need to retroactively alert people, and we should scope alerts narrowly, and there's always the escape hatch of searching.